components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -448,7 +446,7 @@ open class GleanInternalAPI internal constructor () {
     @VisibleForTesting(otherwise = VisibleForTesting.NONE)
     fun enableTestingMode() {
         @Suppress("EXPERIMENTAL_API_USAGE")
-        Dispatchers.API.enableTestingMode()
+        Dispatchers.API.setTestingMode(enabled = true)
```

Comment:
Would it be useful to provide a way for third-parties to also temporarily disable testing mode?

Reply:
I think it would: I filed [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1545759).